collectors/proc.plugin/metadata.yaml

```diff
@@ -5180,3 +5180,138 @@ modules:
                 - name: now
                 - name: max
                 - name: max_design
+  - meta:
+      plugin_name: proc.plugin
+      module_name: /sys/class/drm
+      monitored_instance:
+        name: AMD GPU
+        link: "https://www.amd.com"
+        categories:
+          - data-collection.hardware-devices-and-sensors
+        icon_filename: "amd.png"
+      related_resources:
+        integrations:
+          list: []
+      info_provided_to_referring_integrations:
+        description: ""
+      keywords:
+        - amd
+        - gpu
+        - hardware
+      most_popular: false
+    overview:
+      data_collection:
+        metrics_description: "This integration monitors AMD GPU metrics, such as utilization, clock frequency and memory usage."
+        method_description: ""
```

Comment:
Please, add a method description.

Reply:
What's a method description? None of the other `proc` collectors has one.

Reply:
Let's update meta in a separate PR.